### PR TITLE
Add browser support note

### PR DESCRIPTION
**Pull request by jimmynotjim**
Expand on differences between cfgov-refresh and this project's browser support setups.

## Changes

- Updated contributing with a note on support configuration and delivery

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,6 +167,15 @@ features. For more information visit the [Babel documentation site]
 - https://saucelabs.com/beta/dashboard/tests
 - http://developer.samsung.com/remotetestlab/rtlDeviceList.action#
 
+#### Notes
+
+The CSS and JS files that are generated during the build task are only used
```
Comment (anselmbradford) [inline CONTRIBUTING.md]: Trivial and this is widely used, but earlier in this file it's `JavaScript` and this is the first mention of `JS`. Probably should add `JavaScript (JS)` to the earlier mention.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -167,6 +167,15 @@ features. For more information visit the [Babel documentation site]
...
+for testing. Because this project doesn't necessarily produce a final product
+it is up to the projects that use it to generate and maintain their own browser
+support config. An example is cfgov-refresh generates specific IE 8 and 9
```
Comment (anselmbradford) [inline CONTRIBUTING.md]: `An example…` sentence reads a little awkward. Missing a `, which`?

Comment (ascott1) [reply CONTRIBUTING.md]: How about:

```
An example of this is [cfgov-refresh](https://github.com/cfpb/cfgov-refresh), which generates specific IE 8 and 9 stylesheets, whereas the Capital Framework build task bundles those with the main stylesheet.
```

Comment (anselmbradford) [reply CONTRIBUTING.md]: Since we're talking about the final product, it could read `… [consumerfinance.gov](https://github.com/cfpb/cfgov-refresh) …`

**Comment by jimmynotjim** [comment]
@anselmbradford finally got around to updating this, ready for sign-off.

**Pull request merged**